src/Pester.psd1

```diff
@@ -111,11 +111,14 @@
             LicenseUri   = "https://www.apache.org/licenses/LICENSE-2.0.html"
 
             # Release notes for this particular version of the module
-            ReleaseNotes = 'https://github.com/pester/Pester/releases/tag/5.2.0-alpha3'
+            ReleaseNotes = 'https://github.com/pester/Pester/releases/tag/5.2.0-alpha4'
 
             # Prerelease string of this module
-            Prerelease   = 'alpha3'
+            Prerelease   = 'alpha4'
         }
+
+        # Minimum assembly version required
+        RequiredAssemblyVersion = '5.2.0.0'
```

Comment:
```suggestion
        RequiredAssemblyVersion = '5.2.0'
```

I don't see myself ever specifying anything else than the three pieces.

Reply:
It was your own commit, so okay by me 👍